front/src/applications/operationalStudies/views/ImportTimetableItem.tsx

```diff
@@ -46,20 +52,20 @@ const ImportTrainSchedule = ({
 
   return rollingStocks ? (
     <main className="import-train-schedule">
```

Comment:
We can change the classes as well
```suggestion
    <main className="import-timetable-item">
```

Reply:
Done

Reply:
Done

Reply:
Did you pushed the changes ?

Reply:
Yes